index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>todos</title>
+  </head>
+  <body>
+    <div class="main-wrapper">
+      <header class="todos-header">
+        <h1 class="todos-header-text">todos</h1>
+      </header>
+      <div class="todos-panel">
+        <section class="todos-panel-newtask">
+          <input
+            type="image"
+            class="todo-panel-button-checkall"
+            value="|\/|"
+            aria-label="Select all tasks"
+          />
+          <input
+            type="text"
+            class="todo-panel-newtask-input"
+            placeholder="input new task"
+            aria-label="Add new task"
+            autofocus="autofocus"
+          />
+        </section>
+        <section class="todos-panel-list">
+          <ul class="todos-panel-list tasks">
+            <li class="todo-panel-list-task">
+              <input
+                type="checkbox"
+                class="todo-panel-checkbox"
+                aria-label="Complete task: Task 1"
+              />
+              <span class="todo-panel-list-task-text">Task 1</span>
+              <input
+                type="image"
+                class="todo-panel-button-clear"
+                value="X"
+                aria-label="Delete task: Task 1"
+              />
+            </li>
+            <li class="todo-panel-list-task">
+              <input
+                type="checkbox"
+                class="todo-panel-checkbox"
+                aria-label="Complete task: Task 2"
+              />
+              <span class="todo-panel-list-task-text">Task 2</span>
+              <input
+                type="image"
+                class="todo-panel-button-clear"
+                value="X"
+                aria-label="Delete task: Task 2"
+              />
+            </li>
+            <li class="todo-panel-list-task">
+              <input
+                type="checkbox"
+                class="todo-panel-checkbox"
+                aria-label="Complete task: Task 3"
+              />
+              <span class="todo-panel-list-task-text">Task 3</span>
+              <input
+                type="image"
+                class="todo-panel-button-clear"
+                value="X"
+                aria-label="Delete task: Task 3"
+              />
+            </li>
+          </ul>
+        </section>
+        <section class="todos-panel-actions">
+          <message class="todos-panel-actions-message">3 items left</message>
```

Comment:
A valid custom element name is a sequence of characters name that meets all of the following requirements: name must match the PotentialCustomElementName production:

PotentialCustomElementName ::=
[a-z] (PCENChar)* '-' (PCENChar)*
PCENChar ::=
"-" | "." | [0-9] | "_" | [a-z] | #xB7 | [#xC0-#xD6] | [#xD8-#xF6] | [#xF8-#x37D] | [#x37F-#x1FFF] | [#x200C-#x200D] | [#x203F-#x2040] | [#x2070-#x218F] | [#x2C00-#x2FEF] | [#x3001-#xD7FF] | [#xF900-#xFDCF] | [#xFDF0-#xFFFD] | [#x10000-#xEFFFF]

(http://w3c.github.io/webcomponents/spec/custom/#dfn-custom-element-type)